configure.py

```diff
@@ -475,6 +475,8 @@ def find_ninja():
     'test/boost/double_decker_test',
     'test/boost/duration_test',
     'test/boost/dynamic_bitset_test',
+    'test/boost/encrypted_file_test',
```

Comment:
could you please use "test" as the prefix of commit message's title?

also, might want to correct the typo of "migtrated".

Reply:
done